login.html

```diff
@@ -0,0 +1,203 @@
+            height: auto;
+            margin-bottom: 30px;
+        }
+
+        .input-container {
+            width: 100%;
+            text-align: left;
+            margin-bottom: 20px;
+        }
+
+        .input-container label {
+            display: block;
+            font-size: 1.2em;
+            font-weight: bold;
+            margin-bottom: 5px;
+        }
+
+        .rounded-input, .password-container input {
+            width: 100%;
+            padding: 16px;
+            font-size: 1.2em;
+            border-radius: 30px;
+            background-color: var(--gray100);
+            border: none;
+            box-sizing: border-box;
+        }
+
+        .password-container {
+            display: flex;
+            align-items: center;
+            position: relative;
+        }
+
+        .toggle-password {
+            position: absolute;
+            right: 15px;
+            cursor: pointer;
+            width: 26px;
+            height: 26px;
+        }
+
+        .login-btn {
+            background-color: var(--primary-blue);
+            color: white;
+            padding: 16px;
+            border: none;
+            border-radius: 30px;
+            width: 100%;
+            font-size: 1.3em;
+            cursor: pointer;
+            margin-top: 12px;
+        }
+
+
+        .social-login {
+            width: 100%;
+            max-width: 450px;
+            background-color: #E3F2FD;
+            padding: 15px;
+            border-radius: 10px; 
+            display: flex;
+            align-items: center; 
+            justify-content: center; 
+            box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+            margin-top: 30px;
+        }
+
+        .social-login p {
+            font-size: 1.1em;
+            font-weight: bold;
+            margin: 0;
+            margin-left: 10px; /* Space between icons and text */
+        }
+
+        /* Align social icons on the left */
+        .social-icons {
+            display: flex;
+            gap: 10px;
+            justify-content: flex-start; /* Align icons to the left */
+            align-items: center;
+        }
+
+        /* Social Icon Buttons */
+        .social-icon {
+            width: 35px;
+            height: 35px;
+            border-radius: 50%;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            background-color: white;
+            box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+        }
+
+        /* Adjust logo sizes inside icons */
+        .social-icon img {
+            width: 70%;
+            height: auto;
+        }
+
+
+        .signup-link {
+            font-size: 1.1em;
+            margin-top: 15px;
+        }
+
+
+        @media (max-width: 768px) {
+            .login-container {
+                width: 90%;
+            }
+        }
+
+    </style>
+</head>
+<body>
+    <div class="login-container">
+        <img src="assets/Property 1=sm.png" alt="판다마켓 로고" class="logo">
```

Comment:
❗️ 수정요청
`"판다마켓" 클릭 시 루트 페이지("/")로 이동합니다.(새로고침)` 를 구현하기 위해 해당 이미지 a 태그로 감싸주세요~
index.html의 header 컴포넌트의 로고를 생각하시면 좋을 것 같아요~

```suggestion
        <a href="/"><img src="assets/Property 1=sm.png" alt="판다마켓" class="logo"></a>
```